app/renderer/abr-document.js

```diff
@@ -350,6 +366,65 @@ AbrDocument.prototype = {
         return this.save(path, callback);
     },
 
+    initWatcher: function () {
+        var that = this;
+        var updateTitle = function () {
```

Comment:
From what I understand, `updateWindowTitle()` relies on CodeMirror's state to declare the document as clean or not.

Technically, since the document didn't change in CodeMirror, calling `updateWindowTitle()` here would not add the `*` symbol in the window title. I didn't try it though, so I might be wrong.

In the end, I just wanted a visual feedback for the user without forcing CodeMirror to change its state (maybe with side effects I don't know about).